schemas/JSON/settings/settings.schema.0.2.json

```diff
@@ -96,7 +96,33 @@
       "type": "object",
       "properties": {
         "preferences": { "$ref": "#/definitions/InstallPrefReq" },
-        "requirements": { "$ref": "#/definitions/InstallPrefReq" }
+        "requirements": { "$ref": "#/definitions/InstallPrefReq" },
+        "ignoreWarnings": {
+          "description": "Controls whether blocking warning messages shown to the user during an install or upgrade are ignored",
+          "type": "boolean",
+          "default": false
+        },
+        "PortableAppUserRoot": {
+          "description": "The default root directory where packages are installed to under User scope. Applies to the portable installer type.",
+          "type": "string",
+          "default": "%LOCALAPPDATA%/Microsoft/WinGet/Packages/"
+        },
+        "PortableAppMachineRoot": {
+          "description": "The default root directory where packages are installed to under Machine scope. Applies to the portable installer type.",
+          "type": "string",
+          "default": "%PROGRAMFILES%/WinGet/Packages/"
+        }
+      }
+    },
+    "UninstallBehavior": {
+      "description": "Uninstall settings",
+      "type": "object",
+      "properties": {
+        "purge": {
+          "description": "Controls whether the default behavior for uninstall should remove all files and directories relevant to this package. Only applies to the portable installerType.",
```

Comment:
>Only applies to the portable installerType. [](http://example.com/codeflow?start=144&length=43)

I think "portableAppPurge" will be more clear and consistent with above entries

Reply:
changed to `purgePortableApp` as it's a boolean and should start with a verb